### PR TITLE
[PB-11, PB-18, PB-21] Styles, basic structure, admin pages init

### PR DESCRIPTION
**Pull request by HermanTkachenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 ## Getting Started
 
+1. Install dependencies with npm install or npm i or npm ci
+2. Create .env file in project root with variables from .env.example
+3. Run the project:
+
 ```bash
+# For development purposes
 npm run dev
+npm run lint
```
Comment (SiranWeb) [inline README.md:12]: Remove this line. It's not related to **Run the project**

File: pages/resources.tsx [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import { Resources } from "@/pages";
```
Comment (SiranWeb) [inline pages/resources.tsx:1]: Can you change file path from `pages/resources.tsx` to `pages/resources/index.tsx`?

File: src/shared/components/header.module.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  margin: auto;
+
+  @include breakpoint(xxl, max) {
+    width: 85%;
+  }
+
+  @include breakpoint(md, max) {
+    width: 100%;
+  }
+}
```
Comment (SiranWeb) [inline src/shared/components/header.module.scss:28]: I like that syntax so much, great job ✊

File: src/shared/components/layout/articles/articles.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import styles from "./articles.module.scss";
+
+export const Articles = ({ }) => { // Thats likely incorrect naming, will need to decide it with team
```
Comment (SiranWeb) [inline src/shared/components/layout/articles/articles.tsx:3]: I will take care of it

File: src/shared/components/layout/sidebar/sidebarSection.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import styles from "./sidebarSection.module.scss";
+
+export interface ISidebarSection {
+  section: string;
+  subSections: Array<{
+    id: number;
+    title: string;
+  }>;
```
Comment (SiranWeb) [inline src/shared/components/layout/sidebar/sidebarSection.tsx:8]: It's better to use syntax:
```
{
id: number;
title: string;
}[]
```

File: src/shared/components/layout/topic/topic.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import styles from "./topic.module.scss";
+
+interface ITopic {
+  id: number;
+  title: string;
+  description: string;
+}
+
+export const Topic: React.FC<ITopic[]> = ({ ...topicData }) => {
+  const { title, description } = topicData[0];
```
Comment (SiranWeb) [inline src/shared/components/layout/topic/topic.tsx:10]: Why you pass here array instead of object?

File: src/shared/styles/globals.scss [hunk 7]
```diff
@@ -1,3 +1,18 @@
...
+  --main-background: #fafafa;
+  --main-text-headings: #1e1e24;
+  --main-text: #767474;
+  --main-header-bar: #337ca0;
+  --main-headings: #64afd3;
+
+  // Dark theme
+  // TBD
+
+  /* GLOBAL PAGE CONTAINER STYLES */
```
Comment (SiranWeb) [inline src/shared/styles/globals.scss:16]: Please, remove comments like that. They are pretty useless and litter the code

**Review by SiranWeb — CHANGES_REQUESTED** [review]
Pretty good. Just a few changes. And also pay attention to merge conflicts

File: src/shared/components/layout/topic/topic.tsx (hunk 6 above)
Comment (HermanTkachenko) [reply src/shared/components/layout/topic/topic.tsx:10]: made separated array for topic data, i believe putting info inside sidebarData would make more sense 
should i merge topicData with sidebarData?

Comment (SiranWeb) [reply src/shared/components/layout/topic/topic.tsx:10]: I think yes

**Review by SiranWeb — APPROVED** [review]

**Pull request merged**